src/types/index.tsx

```diff
@@ -0,0 +1,8 @@
+export type CryptoTypes = {
+    id:string,
+    name:string,
+    slug:string,
+    price: string,
+    market_porcentage:string,
+    image:string,
+  };
```

Comment:
I would advice to check the api responses to know the correct type of each so that when you connect data with the API you don't have to adapt the code so much